src/transformers/utils/logging.py

```diff
@@ -213,11 +213,16 @@ def add_handler(handler: logging.Handler) -> None:
 
 
 def remove_handler(handler: logging.Handler) -> None:
-    """removes given handler from the HuggingFace Transformers's root logger."""
+    """Removes given handler from the HuggingFace Transformer's root logger."""
```

Comment:
I think `Transformers` is correct, that's the name of the lib :)

Reply:
Yes :) You're most welcome, @LysandreJik